packages/frameworks/src/frameworks.ts

```diff
@@ -2022,6 +2022,61 @@ export const frameworks = [
     },
     dependency: 'vite',
     getOutputDirName: async () => 'dist',
+    defaultRoutes: [
+      {
+        src: '/public/(.*)',
```

Comment:
I don't think this is correct. This would only work if `vite build` emits a `./dist/public` directory with static assets hashed. Perhaps it should be `/assets/(.*)` instead?

Reply:
Actually the output file is dist for any framework.


<img width="331" alt="Screen Shot 2022-05-13 at 2 43 22 PM" src="https://user-images.githubusercontent.com/10179494/168378864-34c1815a-be43-4472-ac3f-6d64e4b3a080.png">
3.27 PM.png…]()

Reply:
Right, and the routes are relative the the Output Directory (in this case `dist`)

Notice how your screenshot doesn't have a `public` directory.